wiki/History_of_osu!/April_Fools/de.md

```diff
@@ -131,15 +131,15 @@ Das Update wurde von den Spielern allgemein gut angenommen und eine tatsächlich
 
 Am 1. April 2016 wurde [ein Newsbeitrag](https://osu.ppy.sh/home/news/2016-04-01-oculus-rift-to-be-supported-as-an-input-method) veröffentlicht, in dem die Pläne für die Unterstützung der [Oculus Rift](https://de.wikipedia.org/wiki/Oculus_Rift) als eine neue [Eingabemethode](/wiki/Gameplay/Input_device) für osu! angekündigt wurden. In dem von [Evrien](https://osu.ppy.sh/users/791660) verfassten Beitrag wurden viele Zitate aus einem angeblichen Interview mit [peppy](https://osu.ppy.sh/users/2) angeführt, in welchem er seine Gründe für die Ankündigung erklärte und Ideen aufzeigte, wie das Konzept funktionieren könnte.[^osu-vr-news]
 
-In Bezug auf die Art und Weise, wie Spieler die Oculus Rift als Eingabemethode nutzen können, wird im Newspost beschrieben, dass "der Spieler den Cursor in der Ich-Perspektive auf seinem Weg von und zu Hit-Objekten auf dem Bildschirm erlebt (…)" und dass Hit-Objekte getroffen werden, "(…) indem der Spieler vokalähnliche Laute mit seinem Mund erzeugt". Es wurden keine tatsächlichen Änderungen im Spiel vorgenommen, die mit der Verwendung von Oculus Rift oder einem ähnlichen [Gerät mit VR-Unterstützung (Virtuelle Realität)](https://de.wikipedia.org/wiki/Virtuelle_Realit%C3%A4t) zur Steuerung von osu! zusammenhängen.[^osu-vr-news]
+In Bezug auf die Art und Weise, wie Spieler die Oculus Rift als Eingabemethode nutzen können, wird im Newspost beschrieben, dass "der Spieler den Cursor in der Ich-Perspektive auf seinem Weg von und zu Hit-Objekten auf dem Bildschirm erlebt (…)" und dass Hit-Objekte getroffen werden, "(…) indem der Spieler vokalähnliche Laute mit seinem Mund erzeugt".[^translated] Es wurden keine tatsächlichen Änderungen im Spiel vorgenommen, die mit der Verwendung von Oculus Rift oder einem ähnlichen [Gerät mit VR-Unterstützung (Virtuelle Realität)](https://de.wikipedia.org/wiki/Virtuelle_Realit%C3%A4t) zur Steuerung von osu! zusammenhängen.[^osu-vr-news]
 
 *Hinweis: McOsu wird getrennt entwickelt und steht in keiner direkten Verbindung mit osu! oder ppy Pty Ltd.*
```

Comment:
I always found it weird that this line is displayed before McOsu is actually introduced in the paragraph below. Maybe this should be moved down at some point? 🤔

(out of scope here)